Сорокин Владимир

sorovlad opened the pull request.

Commit:
solution

gogoleff commented on index.html:
```diff
@@ -6,6 +6,284 @@
     </head>
     <body>
-        <!-- Здесь прикладывать основное усилие -->
+        <div class="wrapper">
```
Вся галерея вполне сойдёт за `article`

gogoleff commented on index.html:
```diff
@@ -6,6 +6,284 @@
+            <label for="" class="button buttonLeft4"></label>
+
+            <section class="section1">
+                <div class="block">
```
block слишком абстрактно, лучше называть более семантично – `full-image-modal`

gogoleff commented on index.css:
```diff
@@ -1 +1,185 @@
+.buttonRight3,
+.buttonRight4
+{
+    background: url(image/left.png);
```
Такие треугольные стрелки можно сделать и без изображений, свойством `border`

gogoleff commented on index.css:
```diff
@@ -1 +1,185 @@
+#nav1:checked ~ .navigation .nav1,
+#nav2:checked ~ .navigation .nav2,
+#nav3:checked ~ .navigation .nav3,
+#nav4:checked ~ .navigation .nav4
```
Не получится ли объеденить `#nav4:checked ~ .navigation .nav`, чтобы проще было добавлять группы

gogoleff commented on index.css:
```diff
@@ -1 +1,185 @@
+#nav3:checked ~ .images .buttonRight4,
+#nav3:checked ~ .images .buttonLeft2,
+#nav4:checked ~ .images .buttonLeft3,
+#nav4:checked ~ .images .buttonRight1
```
Чтобы добавить ещё группы придётся лезть в css, получиться ли этого избежать?

gogoleff commented on index.css:
```diff
@@ -1 +1,185 @@
+.buttonLeft1,
+.buttonLeft2,
+.buttonLeft3,
+.buttonLeft4
```
Можно создать класс `button-left` и не дублировать эти 4 строки

gogoleff commented:
:star: За решение дополнительных заданий

gogoleff commented:
@sorovlad  Посмотрел! 

Ещё пожелание: при переключение стрелки прыгают вверх-вниз, будет круто этого избежать

Commit:
исправления

убраны id из css, больше нет ограничений на добавление
сделаны кнопки с помощью border(мне не нравятся, тк их область больше
нужной, можно исправить?)
кнопки переключения больше не прыгают, но и не привязаны к середине
картинок

sorovlad commented:
@gogoleff Все починил. От радиокнопок внизу пришлось отказаться, не знаю как реализовать.
P.S. Почему не пятая?

gogoleff commented on index.css:
```diff
@@ -1 +1,145 @@
+.buttonLeft
+{
+    border: 60px solid transparent;
+    border-right: 60px solid #888;
```
:+1:

gogoleff commented:
@sorovlad  Всё здорово, ещё можно `cursor:pointer` навешать на стрелки, но в целом задача принята

gogoleff commented:
:star: За дополнительное задание

gogoleff commented:
:star2: За лаконичное и понятное решение

gogoleff commented:
@sorovlad Пятую проверяю
